File: Python/ModMatShape.py
def ModMatShape(key_ident, i, alpha_len, matrix_len):
    for j in range(matrix_len):
        if j != i:
            key_ident[j, :] = (
                (key_ident[j, :] - key_ident[j][i] * key_ident[i, :]) %
                alpha_len)
    return key_ident

File: Julia/ModMatShape.jl
function ModMatShape(key_ident, i, alpha_len, matrix_len)
    for j in 1:matrix_len
        if j != i
            key_ident[j, :] = mod.(key_ident[j, :] - key_ident[j, i] * key_ident[i, :], alpha_len)
        end
    end
    return key_ident
end